app/javascript/components/miq-about-modal.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux';
+import { combineReducers, bindActionCreators } from 'redux';
+import { AboutModal } from 'patternfly-react';
+
+const SHOW_ABOUT_MODAL = '@@aboutModal/show';
+const HIDE_ABOUT_MODAL = '@@aboutModal/hide';
+const LOAD_ABOUT_MODAL = '@@aboutModal/load';
+
+const showModal = () => (dispatch, getState) => {
+  if (getState().AboutModal && getState().AboutModal.data) {
+    // If the modal has been already rendered, do not request its data twice
+    return dispatch({type: SHOW_ABOUT_MODAL});
+  }
+
+  // Request the modal data from the API
+  return API.get('/api')
+    .then((data) => dispatch({type: LOAD_ABOUT_MODAL, data}))
```

Comment:
need a `catch` block here - you know, just in case...

`.catch(e => reject(e));`

Reply:
Also if you want to avoid the default error handler for miq fetch (big modal with error) and define custom one, you have to add a option in the request that will disable it. You should also add second parameter to the first `then` callback which will throw an error.

Reply:
I don't think we want that @Hyperkid123

Reply:
@AparnaKarve also don't think that I want any kind of handling here, if the API isn't accessible then it's a serious issue...

Reply:
@skateman : the API call failure shoud not be handled here ofc. It should be handled in a single place with a single piece of code. A single error handler for a consistent behavior.

We discussed that with @himdet some time ago (when I noticed there was no global handler). I am not sure if one was added meanwhile.

Anyway, as I wrote, it should not be handled here (in the About modal) nor in this PR.

Reply:
@martinpovolny I got to the same conclusion :+1:

Reply:
The global handler happens by default when calling any `API.*` function, so hopefully nothing special needed for that.